package.json

```diff
@@ -55,7 +55,7 @@
   },
   "lint-staged": {
     "*.{js,ts,tsx}": "eslint --cache --ext .js,.ts,.tsx --max-warnings 0",
-    "*.{js,json,ts,tsx,css,md,mdx}": "prettier --write"
+    "*.{js,json,ts,tsx,css,md,mdx}": "prettier --write ."
```

Comment:
Why are we formatting the entire project every time?

Reply:
I didn't add in the prettier step in lint-staged but just updated it to run properly. I think formatting as a task will help the repo remain formatted.

Reply:
This command (without the `.`) should format any files which are changed though - unless we're missing something. 
I think this version of the command could just go in the pre-commit hook in husky? It's a somewhat bad precedent to set linting the entire codebase - I can imagine a larger modular repo definitely wouldn't want to format 1000s of files in one go!

Reply:
lint-staged is the precommit husky hook  but yeah I do agree it can get intensive. I re-read the docs and I think I misread them so I will revert